<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketOrderDlg1</class>
 <widget class="QDialog" name="TicketOrderDlg1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Customer</string>
        </property>
        <property name="buddy">
         <cstring>customerLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;When</string>
        </property>
        <property name="buddy">
         <cstring>whenDateTimeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QDoubleSpinBox" name="priceDoubleSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string>$ </string>
        </property>
        <property name="maximum">
         <double>5000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3" rowspan="2">
       <widget class="QSpinBox" name="quantitySpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="customerLineEdit"/>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Price</string>
        </property>
        <property name="buddy">
         <cstring>priceDoubleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Quantity</string>
        </property>
        <property name="buddy">
         <cstring>quantitySpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="4" rowspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Amount</string>
        </property>
        <property name="buddy">
         <cstring>amountLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="5" rowspan="2">
       <widget class="QLineEdit" name="amountLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QDateTimeEdit" name="whenDateTimeEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TicketOrderDlg1</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TicketOrderDlg1</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
